### DISCUSSION TITLE
TOML file render table after comment as invalid format

### DISCUSSION THREAD

**Discussion by waynexia** [community/community#110255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

An example TOML file: https://github.com/waynexia/greptimedb/blob/83ebdb5f358469c570ce1b91d4e16980f65a3586/config/frontend-example.toml

Some table is rendered as invalid format (with red background). Screenshot:

<img width="584" alt="image" src="https://github.com/community/community/assets/15380403/a22b10ac-0af5-482b-ab74-86a101651fc5">

Table after comment is not forbidden in TOML spec: https://toml.io/en/v1.0.0#comment

And in its document it has some example that puts comment before table, e.g.:
 
<img width="856" alt="image" src="https://github.com/community/community/assets/15380403/dbfad063-05c5-46e3-97df-5c06442cd2bd">

**Comment by sanjay-ar** [comment]
little complex....
